src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -158,7 +157,14 @@ class WorkspaceInvitePage extends React.Component {
         });
         indexOffset += personalDetailsWithoutSelected.length;
 
-        if (hasUnselectedUserToInvite) {
+        const hasUnselectedUserToInvite = this.state.userToInvite && !_.contains(selectedLogins, this.state.userToInvite.login);
+        const hideUserToInvite =
+            this.state.userToInvite &&
+            _.find(personalDetailsWithoutSelected, ({login}) => {
+                const phone = `${this.state.userToInvite.login}${CONST.SMS.DOMAIN}`;
```

Comment:
Do you mean only to prevent this on phone numbers? Could we use `OptionsListUtils.addSMSDomainIfPhoneNumber` instead?

Reply:
To fix duplicate when selecting user with phone number I switched to personalDetails instead of personalDetailsWithoutSelected.

Initially this was an issue with phone numbers only.

Reused OptionsListUtils.addSMSDomainIfPhoneNumber.

Thanks!